<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>versions_new_view_controller</class>
    <widget class="QDialog" name="versions_new_view_controller">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>418</width>
                <height>340</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="label">
                    <property name="text">
                        <string>Version</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLineEdit" name="txt_ver_name"/>
            </item>
            <item>
                <widget class="QLabel" name="label_2">
                    <property name="text">
                        <string>File</string>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QLineEdit" name="txt_file"/>
                    </item>
                    <item>
                        <widget class="QPushButton" name="btn_chose_file">
                            <property name="text">
                                <string>Chose File</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QLabel" name="label_3">
                    <property name="text">
                        <string>Desc</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QPlainTextEdit" name="txt_ver_desc">
                    <property name="minimumSize">
                        <size>
                            <width>0</width>
                            <height>0</height>
                        </size>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                        <spacer name="horizontalSpacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QCheckBox" name="cb_head">
                            <property name="text">
                                <string>HEAD</string>
                            </property>
                            <property name="checked">
                                <bool>true</bool>
                            </property>
                            <property name="tristate">
                                <bool>false</bool>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="button_box">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
